src/util/projects.tsx

```diff
@@ -12,9 +14,19 @@ export const projectSubpages = [
 
 export const getSubpageFromUrl = (url: string) => {
   const parts = url.split("/");
-  if (projectSubpages.includes(parts[4])) {
-    return parts[4];
+
+  const mainSubpage = parts[4];
+  const tabSubpage = parts[5];
+  const slugifiedStorageTabs = STORAGE_TABS.map((tab) => slugify(tab));
```

Comment:
I don't like that we leak the slugify into this component, Can we define a `export const STORAGE_TAB_URLS = TABS.map((tab) => slugify(tab));` in `Storage.tsx` and then use it here directly?
Using the `STORAGE_TAB_URLS` could also simplify the `Storage.tsx` code a bit.

Reply:
Sure, good idea.